app/scripts/modules/core/src/presentation/forms/fields/TextField.tsx

```diff
@@ -10,14 +11,26 @@ export class TextField extends React.Component<ITextFieldProps> {
   public static defaultProps: Partial<IFieldProps> = { FieldLayout: BasicLayout };
 
   public render() {
-    const { label, help, error, warning, preview, actions, ...rest } = this.props;
-    const { FieldLayout, value, onChange, required, ...inputProps } = rest;
+    const { label, help, touched, error, warning, preview, actions, ...rest } = this.props;
+    const { FieldLayout, value, onChange, onBlur, required, ...inputProps } = rest;
+
+    const className = classNames({
```

Comment:
using these for now until consolidated forms design is complete and the style guide is updated